src/main/java/seedu/address/model/userimage/FilePath.java

```diff
@@ -0,0 +1,55 @@
+package seedu.address.model.userimage;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.io.File;
+
+/**
+ * Creates a new FilePath object from a valid filePath
+ */
```

Comment:
I think this should be for the constructor instead?

Reply:
Thanks for the reminder. Javadoc has been changed to make it clearer
```
/**
 * Represents a file path to a file in the user's file system.
 * Guarantees: immutable; is valid as declared in {@link #isValidFilePath(String filepath)}
 */
```